CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [0.0.2]
```

Comment:
the changelog is generated automatically, see this example for what you should add to a PR: https://github.com/webview/webview_rust/pull/30/files#diff-927581e3f911d71c928f5d4267678046

basically you add a markdown file to the .changes folder, and say what kind of package should be updated, and the version change (major, minor, patch) needed, along with a summary.